plugin/external/pluginEntry.go

```diff
@@ -535,6 +614,19 @@ func (e *pluginEntry) Stream(ctx context.Context) (io.ReadCloser, error) {
 }
 
 func (e *pluginEntry) Exec(ctx context.Context, cmd string, args []string, opts plugin.ExecOptions) (plugin.ExecCommand, error) {
+	if result := e.methods["exec"].tupleValue; result != nil {
+		impl, ok := result.(execImpl)
```

Comment:
This validation feels redundant but maybe that's just me.

Reply:
Will be after a refactor a bit.

Reply:
Ah yeah, I understand now. Dropped checking the cast.